packages/apps-v5/src/commands/ps/scale.js

```diff
@@ -12,7 +12,37 @@ https://devcenter.heroku.com/articles/procfile`)
 async function run(context, heroku) {
   let app = context.app
 
+  // will remove this flag once we have
+  // successfully launched larger dyno sizes
+  let isLargerDyno = false
+  const feature = await heroku.get('/account/features/frontend-larger-dynos')
+
   function parse(args) {
+    // will remove this flag once we have
+    // successfully launched larger dyno sizes
+    const featureNotEnabled = !feature.enabled
```

Comment:
Is this necessary? Or could the if statement below just use the feature flag itself?

Reply:
I was going for better readability. However, yes, I can update this `if` statement.